src/codegen/transformations.lisp

```diff
@@ -37,6 +37,28 @@
   (type     'node                     :type node-subtype :read-only t)
   (function (util:required 'function) :type function     :read-only t))
 
+(defmacro action ((when type &rest args) &body body)
```

Comment:
This macro seems to generate lots of warnings about the node class names being undefined...